src/main/java/org/broadinstitute/hellbender/tools/variantdb/arrays/ArrayExtractCohort.java

```diff
@@ -171,7 +203,11 @@ protected void onStartup() {
             probeIdMap = ProbeInfo.getProbeIdMapFromExport(probeCsvExportFile);
         }
 
-
+        // if we have a qcMetrics table, augment the probeInfo map with that information
+        Map<Long, ProbeQcMetrics> probeQcMetricsMap = null;
+        if (qcMetricsTableName != null) {
+            probeQcMetricsMap = ProbeQcMetrics.getProbeQcMetricsWithStorageAPI(qcMetricsTableName);
```

Comment:
I don't see this class, did you commit `ProbeQcMetrics`?

Reply:
I did now 😅